docs/development.md

```diff
@@ -1,119 +1,125 @@
 # Faasm Development
```

Comment:
Worth looking at the final version rather than the diff: https://github.com/faasm/faasm/blob/func-move/docs/development.md

Reply:
Looking :new: !